app/views/response/_response_analysis.html.erb

```diff
@@ -0,0 +1,415 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Convert JSON Data to HTML Table</title>
```

Comment:
since you used something you didn't write here, please add a comment with where this comes from so future developers can maintain the code easier